include/deal.II/grid/grid_tools.h

```diff
@@ -606,6 +606,26 @@ namespace GridTools
   /*@{*/
 
   /**
+   * Given a @p cache and a list of @p points create the quadrature rules. This
+   * function returns a tuple containing the following elements:
+   * - The first ( get<0>) is a vector of the cells containing at least one of the @p points
+   * - The second, call it qpoints, is a vector of vectors. Each entry contains the quadrature points contained in the cell fo cells
```

Comment:
do you mean "vector of cells" at the end of the sentence, instead of "cell of cells"?

Also, why call them "quadrature points"? I don't see quadrature happening here, it's just a bunch of points :-)

Reply:
Yes, the name it's more of a relic of the first version...